hnix-store-core/src/System/Nix/Store/Types.hs

```diff
@@ -1,17 +1,11 @@
+-- | TODO rename module
```

Comment:
Split into `RepairMode.hs` and `PathFilter.hs`? Not sure if worth just for these two.